website/static/js/pages/share-embed-page.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+var pym = require('pym.js');
+
+var registration_url = window.contextVars.share.registration_url;
```

Comment:
> Use lowerCamelCase for variables, properties and function names

 https://github.com/felixge/node-style-guide#use-lowercamelcase-for-variables-properties-and-function-names via https://cosdev.readthedocs.org/en/latest/style_guides/javascript.html#style